debian/kano-updater.postinst

```diff
@@ -46,6 +46,9 @@ case "$1" in
             grep -m 1 '^deb http://.*.kano.me/archive/' $KANO_SOURCES | \
                 sed -r "s|^deb http://([a-z]*).kano.me.*$|deb http://\1.kano.me/archive/ release-urgent main|" >> $KANO_SOURCES
 
+        # Enable systemd services and timers.
+        systemctl enable apt-autoclean-autoremove.service
+        systemctl enable daily-cleanup.timer
```

Comment:
Also `&& systemctl start apt-autoclean-autoremove.service`? Same for timer

Reply:
For most, this will be installed via the Updater in an update. Starting the them would actually trigger and execution when there's going to be a reboot immediately after anyway. I don't think it's needed here.